tfx/components/trainer/fn_args_utils.py

```diff
@@ -59,14 +59,14 @@ def get_common_fn_args(input_dict: Dict[Text, List[types.Artifact]],
                        working_dir: Text = None) -> FnArgs:
   """Get common args of training and tuning."""
   train_files = [
-      io_utils.all_files_pattern(
-          artifact_utils.get_split_uri(input_dict[constants.EXAMPLES_KEY],
-                                       'train'))
+      io_utils.all_files_pattern(uri) for uri in
```

Comment:
could you check the api used in module file allows multiple file patterns?

(in case it only support single pattern or multiple specific files)

Reply:
`trainer_module.py` uses `tf.data.experimental.make_batched_features_dataset()` to process a list of input file names. It accepts a list of patterns 

https://www.tensorflow.org/api_docs/python/tf/data/experimental/make_batched_features_dataset